src/k8s-engine/wsl.ts

```diff
@@ -827,8 +827,10 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
    * Install helper tools for WSL (nerdctl integration).
    */
   protected async installWSLHelpers() {
-    await this.runInstallScript(INSTALL_WSL_HELPERS_SCRIPT,
-      'install-wsl-helpers', await this.wslify(resources.get('linux', 'bin', 'nerdctl-stub')));
+    const fwdSlashNerdctlPath = path.join(paths.resources, 'linux', 'bin', 'nerdctl-stub');
```

Comment:
Doesn't this path actually contain *back*slashes? (That is, [`\`](https://www.fileformat.info/info/unicode/char/005c/index.htm))  Either way, a better name would be `windowsNerdctlPath`.